src/components/Slider2/Slider2.js

```diff
@@ -0,0 +1,101 @@
+import memoize from 'lodash/memoize';
+import times from 'lodash/times';
+import range from 'lodash/range';
+import has from 'lodash/has';
+
+import React, {Component, PropTypes} from 'react';
+import Rheostat from 'rheostat';
+import cx from 'classnames';
+
+import Pit from './Pit.js';
+
+import autoHideContainerHOC from '../../decorators/autoHideContainer.js';
+import headerFooterHOC from '../../decorators/headerFooter.js';
+
+const getCacheKey = obj => Object.keys(obj).map(key => `${key}:${obj[key]}`).join('|');
+
+class RawSlider extends Component {
+
+  propTypes = {
+    refine: PropTypes.func.isRequired,
+    min: PropTypes.number.isRequired,
+    max: PropTypes.number.isRequired,
+    values: PropTypes.arrayOf(PropTypes.number).isRequired,
+    pips: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.object,
+    ]),
+    step: PropTypes.number.isRequired,
+    tooltips: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({format: PropTypes.func.isRequired}),
+    ]),
+  }
+
+  handleChange = ({values}) => {
+    const {refine} = this.props;
+    refine(values);
+  }
+
+  // creates an array number where to display a pit point on the slider
+  computeDefaultPitPoints = memoize(({min, max}) => {
```

Comment:
Works for me, during my tests I was using the same min/max values so found it relevant